backend/orchestrator/libs/orchestratorTable.ts

```diff
@@ -1,19 +1,17 @@
 import DynamoDB from 'aws-sdk/clients/dynamodb';
 import { Table } from 'dynamodb-toolbox';
 
-import { getEnvVariable } from '@swarmion/serverless-helpers';
+import { PARTITION_KEY, SORT_KEY } from 'resources/dynamodb';
 
-import { PARTITION_KEY, SORT_KEY } from '../resources/dynamodb';
+import { ORCHESTRATOR_TABLE_NAME } from './config/dynamodb';
 
 const DocumentClient = new DynamoDB.DocumentClient();
 
-const OrchestratorTable = new Table({
-  name: getEnvVariable('ORCHESTRATOR_TABLE_NAME'),
+export const OrchestratorTable = new Table({
+  name: ORCHESTRATOR_TABLE_NAME,
   partitionKey: PARTITION_KEY,
   sortKey: SORT_KEY,
   autoExecute: true,
   autoParse: true,
   DocumentClient,
 });
```

Comment:
We can leave the `default` export and rename the file into `OrchestratorTable.ts` (with a capital O).
The convention is when a file is exporting is single value, we use a default export with the same name as the file.